src/Microsoft.AspNetCore.Mvc.Core/ApiExplorer/UnknownApiDescriptionProfile.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    public class UnknownApiDescriptionProfile : DefaultApiDescriptionProfile
```

Comment:
The ultimate fallback to add a 'default' problemdetails

Reply:
`CatchAllApiDescriptionProfile`? Since that's a term users might be familiar with.